project.yaml

```diff
@@ -992,7 +1011,7 @@ Layer:
                 foot,
                 bicycle,
                 tracktype,
-                'null',
+                'null' AS surface, -- Should be a SQL NULL?
```

Comment:
sent from a phone

> Il giorno 15 giu 2016, alle ore 12:45, Mateusz Konieczny <notifications@github.com> ha scritto:
> 
> on the other hand querying some other tags like tracktype, bicycle, foot for railways is also rather pointless so there is no good reason to do the same for surface.


maybe this is (or was) to cater for some railway=abandoned that now are tracks, or footways or cycleways?